game/lib/details_page/details_page.dart

```diff
@@ -152,7 +153,7 @@ class _DetailsPageWidgetState extends State<DetailsPageWidget> {
                         Navigator.push(
                             context,
                             MaterialPageRoute(
-                                builder: (context) => BottomNavBar()));
+                                builder: (context) => JourneysPage()));
```

Comment:
What's the purpose of removing the bottom nav bar?

Reply:
Unrelated to my PR